### ISSUE TITLE
how to use another reid model

### ISSUE THREAD

**Issue by szh91** [ifzhang/FairMOT#319]

**Comment by PLester** [comment]
Did you mean using a separate REID model?

**Comment by szh91** [comment]
yes, If I want to use a separate REID model, how to do it

**Comment by PLester** [comment]
@szh91 You can refer to DeepSort (the postprocessing of Fair is just similar to DeepSort), and replace the id embedding with other ReID features, such as MGN (refer to https://github.com/KeyForce/Cascade-RCNN-Tracking).

**Comment by szh91** [comment]
yes, the id embedding modul is in ./src/tracker/multitracker.py？
